challenge3/webapp/caesaris/app/lib/util.rb

```diff
@@ -0,0 +1,51 @@
+require "json"
+
+class Util
+  class << self
+    def form_ransack_params(params)
+      param_org = params
+      param_res = {}
+      if param_org.instance_of?(String)
+        param_org = JSON.parse(param_org)
+      end
+      param_org.each do |key, val|
+        key = key.to_s.underscore
+        if (key == 's' || key == 'sorts') && val.instance_of?(String)
+          param_res[key] = val.split(/\s+/).map(&:underscore).join(' ').strip
```

Comment:
```suggestion
          param_res[key] = val.squish.underscore
```
https://railsguides.jp/active_support_core_extensions.html#squish